.vscode/launch.json

```diff
@@ -39,7 +39,7 @@
         "--target", 
         "lib/main_staging.dart",
         "--dart-define",
-        "AMPLITUDE_KEY=ceb9aaa139ac6028aa34166d6f57923e",
+        "AMPLITUDE_KEY=f4fa9ab88de04c56a346aaa36c172a9a",
```

Comment:
This shouldn't have been added to the secrets?

Reply:
They were already commited in git, but prod vs dev was switched.